extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -394,7 +398,7 @@ class AmpAd extends AMP.BaseElement {
         this.toggleFallback(true);
       }
       // Remove the iframe only if it is not the master.
-      if (this.iframe_.name.indexOf('_master') == -1) {
+      if (this.iframe_ && this.iframe_.name.indexOf('_master') == -1) {
```

Comment:
This just happens if we killed the iframe in the last animation frame. I think this is exactly correct.

One option would be to short circuit the entire callback by `if (!this.iframe_)` since nothing here makes sense without it.